py/aou_workbench_client/concepts.py

```diff
@@ -0,0 +1,103 @@
+from aou_workbench_client.auth import get_authenticated_swagger_client
+from aou_workbench_client.config import all_of_us_config
+from aou_workbench_client.swagger_client.apis.concepts_api import ConceptsApi
+from aou_workbench_client.swagger_client.models.domain import Domain
+from aou_workbench_client.swagger_client.models.search_concepts_request import SearchConceptsRequest
+from aou_workbench_client.swagger_client.models.standard_concept_filter import StandardConceptFilter
+from ipywidgets import interactive
+import pandas as pd
+
+from IPython.display import display, HTML
+
+_DOMAIN_DICT = { '': None,
+    'Observation': Domain.OBSERVATION,
+    'Procedure': Domain.PROCEDURE,
+    'Drug': Domain.DRUG,
+    'Condition': Domain.CONDITION,
+    'Measurement': Domain.MEASUREMENT,
+    'Device': Domain.DEVICE,
+    'Race': Domain.RACE,
+    'Gender': Domain.GENDER,
+    'Ethnicity': Domain.ETHNICITY 
+}
+
+_STANDARD_CONCEPT_FILTER_DICT = { '': StandardConceptFilter.ALL_CONCEPTS,
+    'Standard concepts': StandardConceptFilter.STANDARD_CONCEPTS,
+    'Non-standard concepts': StandardConceptFilter.NON_STANDARD_CONCEPTS
+}
+
+# Common vocabularies. (There are others in the data but they don't get much
+# use.)
+_VOCAB_IDS = [ '', 
+    'ATC',    
+    'CPT4',
+    'DRG',
+    'HCPCS',
+    'ICD10CM',
+    'ICD10PCS',
+    'ICD9CM',
+    'ICD9Proc',
+    'ISBT',
+    'ISBT Attribute',
+    'LOINC',
+    'Multum',
+    'NDC',
+    'NDFRT',
+    'PPI',
+    'RxNorm',
+    'RxNorm Extension',
+    'SNOMED',
+    'SPL',
+    'VA Product'
+  ]
+
+_VOCAB_DICT = {id: id for id in _VOCAB_IDS}
+
+def search_concepts(request):
+  client = get_authenticated_swagger_client()
+  concepts_api = ConceptsApi(api_client=client)
+  response = concepts_api.search_concepts(all_of_us_config.workspace_namespace,
+                                          all_of_us_config.workspace_id,
+                                          request=request)
+  return response.items
+
+def get_concept_dict(concept):
+  return { "ID": concept.concept_id,
```

Comment:
The goal here is not to have users interact with the underlying objects that produced this table... it's to display a widget. The user will then copy/paste a concept ID out of here. (Or maybe in future we'll have the ability to generate code based on their selections.) I'd prefer to have nicely readable headers when displaying actual UI widgets like this.

(My two cents. If anybody feels strongly otherwise, I'll change it.)